pkg/k8s/apis/cilium.io/v2/types.go

```diff
@@ -269,6 +269,170 @@ type CiliumNetworkPolicyList struct {
 // +genclient
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
```

Comment:
add this and generate the code again. This type of CiliumNetworkPolicy won't be namespaced.
```
// +genclient:nonNamespaced
```